stories/coverage.fmf

```diff
@@ -0,0 +1,88 @@
+story:
+    As a QE I want to document the test coverage so it can be easily tracked
+    which requirement is covered by which test case and vice-versa.
+
+/notes:
+    summary:
+        Story notes.
+    story:
+        As a product owner I want to add additional content to the requirements,
+        like notes or context, so extra information can be provided.
+    description:
+        There should be a parameter that has no strict requirement and
+        can be used to provide any extra information that may be necessary.
+    example:
+        - 'story:'
+        - '     As an admin I want product A doing B so I can use it for C.'
+        - 'notes:'
+        - '     The feature should help our customer to achieve D.'
+    priority: low
+
+/priority:
+    summary:
+        Story priority.
+    story:
+        As a product owner I want to prioritize the requirements
+        so the importance can be stated.
+    description:
+        Priority is an important attribute which helps to readers to understand
+        the importance of the story.
+    example:
+        - 'story:'
+        - '     As an admin I want product A doing B so I can use it for C.'
+        - 'priority: high'
+    priority: high
+
+/relevancy:
+    summary:
+        Story relevancy.
+    story:
+        As a product owner I want to to connect requirements with products
+        or versions so I can filter out items relevant (or not relevant)
+        for a particular product.
+    description:
+        Relevancy is an easy way to bind stories to particular product(s).
+    example:
+        - 'story:'
+        - '     As an admin I want product A doing B so I can use it for C.'
+        - 'relevancy: "distro < fedora-28: False"'
+
+/reference-repo:
+    summary:
+        Referencing remote repositories.
+    story:
+        As a QE I want to reference not only local repositories so requirements
+        can be covered by tests cases from a different repository.
+    description:
+        It should be possible to used full FMF Identifier
+        in the tested, implemented, documented story parameters.
+    example:
+        - 'tested:'
+        - '     url: https://github.com/psss/fmf'
+        - '     ref: master'
+        - '     name: /tests/basic/repo'
+    priority: medium
+
+/filter-repo:
+    summary:
+        Filtering based on FMF identifiers.
+    story:
+        As a QE I want to filter based on reference identifiers
+        so I can learn where the tests are implemented/maintained.
+    description:
+        There should be an easy way to filter based on FMF Identifiers.
+    example:
+        - 'tmt story ls --filter "url:https://gitlab/project"'
+        - 'tmt story ls --filter "url:https://gitlab/project & ref:master name:/tests/smoke"'
+    priority: medium
+
+/filter-tags:
```

Comment:
Instead of connecting `filter-repo` using a dash I'd recommend to use a structure:

    /filter:
        /repo:
            summary: ...
            story: ...

        /tags:
            summary: ...
            story: ...